tools/scripts/calculate-commands.js

```diff
@@ -1,10 +1,17 @@
 const execSync = require('child_process').execSync;
 const isMaster = process.argv[2] === 'False';
```

Comment:
`isMaster` is always `false` as `process.argv[2]` will contain the value of `CI_PIPELINE_SOURCE` environment variable and not `False` or `True`.